frontend/main.js

```diff
@@ -57,13 +58,16 @@ async function startApp () {
   // this is to ensure compatibility between frontend and test/backend.test.js
   sbp('okTurtles.data/set', 'API_URL', window.location.origin)
 
-  // TODO: handle any socket errors!
-  createWebSocket(sbp('okTurtles.data/get', 'API_URL'), {
-    // TODO: verify these are good defaults
-    timeout: 3000,
-    strategy: ['disconnect', 'online', 'timeout']
-  })
-
+  sbp('okTurtles.data/set', PUBSUB_INSTANCE, createGIPubSubClient(
+    sbp('okTurtles.data/get', 'API_URL'), {
+      // TODO: verify these are good defaults
+      // Temporarily disable automatic reconnection until it's robust enough.
+      reconnectOnDisconnection: false,
+      reconnectOnOnline: false,
+      reconnectOnTimeout: false,
+      timeout: 3000
```

Comment:
Only `reconnectOnTimeout` is implemented for now, but I'd like to implement the other ones too.

Reply:
OK yeah, for full socket support it would be good to have the equivalent of the options we were using before working, e.g. `strategy: ['disconnect', 'online', 'timeout']`, which I think amounts to the ability for the client to reconnect automatically in the event the internet goes out, or the server goes down briefly, etc.